lib/livingstyleguide/tilt_template.rb

```diff
@@ -85,6 +85,10 @@ def find_root_path
           break
         end
         path = File.expand_path('..', path)
+        if path === '/'
+          puts 'No GemFile or .git found. Compiling anyway.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Do not write to stdout. Use Rails' logger if you want to log.